reactor-core-micrometer/src/main/java/reactor/core/observability/micrometer/MicrometerObservationListener.java

```diff
@@ -73,7 +79,7 @@
 		//creation of the listener matches subscription (Publisher.subscribe(Subscriber) / doFirst)
 		//while doOnSubscription matches the moment where the Publisher acknowledges said subscription
 		subscribeToTerminalObservation = Observation.createNotStarted(
-			configuration.sequenceName + OBSERVATION_FLOW,
+			configuration.sequenceName + OBSERVATION_SUFFIX_FOR_METRICS,
```

Comment:
Why are we adding the `.observation` suffix?

Reply:
good point, at least when `sequenceName` is something actively defined by the user.

my initial thought was that for metrics it could be useful, but if they use metrics they should choose the sequenceName accordingly...

 after thinking about it a bit more, I'm going to keep the derivation of the name from the `name()` operator for now, which means that we still need some sort of default name for the case where none has been explicitly provided. I'll change this accordingly so that here `configuration.sequenceName` is either just the `name()` defined by users or the generic fallback